CHANGELOG.md

```diff
@@ -109,6 +109,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
   - Set/clear/append/rename fields: available through Edit menu
   - Manage keywords: available through Edit menu
   - Copy linked files to folder: available through File menu
+  - Add/move/remove from group: removed completely
```

Comment:
maybe you could add " - available using group interface" as there are still the "add/remove selected entries to this group" options available

Reply:
Good suggestion.